receiver/filelogreceiver/README.md

```diff
@@ -153,4 +153,43 @@ The above configuration will read logs from the "simple.log" file. Some examples
 2023-06-20 12:50:00 DEBUG This is a test debug message
 ```
 
+## Fault tolerance, recovery and proper offset tracking
 
+File offset tracking is quite crucial to ensure the quality of this receiver.
+Filelog receiver has no native support for file offset tracking however this can be achieved by using
+the proper `extentions`.
```

Comment:
I don't think it's accurate to say it has no native support for file offset tracking just because an extension is required. In fact, the component is heavily designed around this need for offset tracking. The fact that an extension is required is simply to allow for flexibility in the mechanism of storage.

I suggest replacing this with most/all of the description of the `storage` parameter.